rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1649,17 +1649,54 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 	case 0xc0000007: // cellFsArcadeHddSerialNumber
 	{
-		break;
+		const auto arg = vm::static_ptr_cast<lv2_file_c000007>(_arg);
+
+		// No idea what these are meant to be
+		std::string unk1 = "";
```

Comment:
The e = "" is redundant

Reply:
I think it's about the meaning it tries to convey - that the correct strings are unknown.

Reply:
conatexpr std::string_view should be used here btw.